mappings/net/minecraft/world/biome/source/BiomeAccessor.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_4543 net/minecraft/world/biome/source/BiomeAccessor
+	CLASS class_4544 Storage
+		METHOD method_16359 accessBiome (III)Lnet/minecraft/class_1959;
```

Comment:
Given this function is on the storage, I think it could just be `getBiome`?

Reply:
No, otherwise people will confuse it as getting biome for block xyz because this method is in world as well (present together with the actual one that gets the biome for blockpos)

Reply:
Ah, I see. `getStorageBiome`?

Reply:
updated